server/webserver.go

```diff
@@ -69,6 +69,10 @@ func (s *LivepeerServer) StartCliWebserver(bindAddr string) {
 func (s *LivepeerServer) cliWebServerHandlers(bindAddr string) *http.ServeMux {
 	mux := http.NewServeMux()
 
+	mux.HandleFunc("/live/", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
is `live` the most appropriate name for this endpoint?

how about `livesegs` ? `livesegment` ? `seglive` ?I think `live` is going to get confusing later on

Reply:
Open to other names. What's confusing about `/live/` ?

I like the brevity of `/live/` though, and can see a path towards us supporting non-segmented transcoding under this API; see the "Non-Segmented Transcodes" section.

Reply:
Personally, I believe that we will discover, in the future, that this is not how we want to transmit live streams.   At that point, we probably won't remove this endpoint, but we'll add another live endpoint, and what will it be called?  As such, the name `live` is very confusing.

Reply:
@mk-livepeer are you referring to your comments on using the object store as a transport mechanism for the segment? Or an entirely different approach to transmitting live streams?

Reply:
I'm not talking about any one solution.  I am saying that "live" is too generic a name for this endpoint, as we are likely to end up having multiple ingest methods.

Reply:
...reflecting on this, I actually really like that video ingress is at `/live` and video egress is at `/stream`. Feels consistent.